feature-libs/organization/administration/components/budget/form/budget-form.component.html

```diff
@@ -34,15 +34,17 @@
       }}</span>
       <cx-date-picker
         [control]="form.get('startDate')"
-        [max]="form.get('endDate')"
+        [max]="form.get('endDate').value"
+        (update)="form.get('endDate').updateValueAndValidity()"
       ></cx-date-picker>
     </label>
 
     <label>
       <span class="label-content">{{ 'orgBudget.endDate' | cxTranslate }}</span>
       <cx-date-picker
         [control]="form.get('endDate')"
-        [min]="form.get('startDate')"
+        [min]="form.get('startDate').value"
+        (update)="form.get('startDate').updateValueAndValidity()"
```

Comment:
Why not keeping the `updateValueAndValidity` in the date picker logic? We had this in the directive, and it makes the date picker easier to use. It will be very hard to know for future date pickers that have min/max relation, that you need to trigger this updateValueAndValidity yourself. 

If we pass in the min/max form control, we can manage this automatically.

Reply:
In my case if someone uses 2 datepickers and would like to make them reactive, he must obvoiusly add this co-validation.

> It makes the date picker easier to use.

yes, but only in case when he uses 2 date pickers (then we should have something like DateRangeComponent)

But for use only one datepicker with min/max attributes, user must play with workarounds (fake date fields with value) 😂

Reply:
Agreed, good point actually.

Reply:
let's go with your solution